torchinductor/config.py

```diff
@@ -28,6 +28,9 @@
 # codegen benchmark harness
 benchmark_harness = True
 
+# autotune kernels
+autotune = False
```

Comment:
perhaps we should make this more specific to conv

Reply:
Remove this autotune